standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/PersistenceManagerProvider.java

```diff
@@ -160,14 +162,14 @@ public static void updatePmfProperties(Configuration conf) {
     boolean readLockAcquired = true;
     try {
       // if pmf properties change, need to update, release read lock and take write lock
-      if (prop == null || pmf == null || !propsFromConf.equals(prop)) {
+      if (pmf == null || !propsFromConf.equals(prop)) {
```

Comment:
Why is this change?

Reply:
unnecessary null check, `!propsFromConf.equals(prop)` covers it